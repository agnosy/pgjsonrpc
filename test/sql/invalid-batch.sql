\set ECHO none
BEGIN;
\t
\i sql/pgjsonrpc.sql
\set ECHO all

-- You should write your tests

SELECT jsonrpc.execute('[1]');
SELECT jsonrpc.execute('[1,2,3]');

ROLLBACK;
