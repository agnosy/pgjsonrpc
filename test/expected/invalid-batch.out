\set ECHO none
-- You should write your tests
SELECT jsonrpc.execute('[1]');
 [{"id": null, "error": {"code": -32600, "message": "Invalid Request"}, "jsonrpc": "2.0"}]

SELECT jsonrpc.execute('[1,2,3]');
 [{"id": null, "error": {"code": -32600, "message": "Invalid Request"}, "jsonrpc": "2.0"}, {"id": null, "error": {"code": -32600, "message": "Invalid Request"}, "jsonrpc": "2.0"}, {"id": null, "error": {"code": -32600, "message": "Invalid Request"}, "jsonrpc": "2.0"}]

ROLLBACK;
